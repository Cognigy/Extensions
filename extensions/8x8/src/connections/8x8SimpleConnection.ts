import type { IConnectionSchema } from '@cognigy/extension-tools';

export const simpleConnection: IConnectionSchema = {
  type: 'eightbyeightsimple',
  label: '8x8',
  fields: [
    { fieldName: 'apiKey' },
    { fieldName: 'clusterBaseUrl' },
    { fieldName: 'tenantId' }
  ]
};

export interface I8x8SimpleConnection {
  apiKey: string
  clusterBaseUrl: string
  tenantId: string
}
