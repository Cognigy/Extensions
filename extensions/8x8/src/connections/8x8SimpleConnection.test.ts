import { simpleConnection } from './8x8SimpleConnection';

describe('8x8SimpleConnection', () => {
  it('should match snapshot', () => {
    expect(simpleConnection).toMatchSnapshot();
  });
});
