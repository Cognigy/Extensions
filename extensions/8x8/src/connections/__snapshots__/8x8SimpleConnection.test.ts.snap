// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`8x8SimpleConnection should match snapshot 1`] = `
{
  "fields": [
    {
      "fieldName": "apiKey",
    },
    {
      "fieldName": "clusterBaseUrl",
    },
    {
      "fieldName": "tenantId",
    },
  ],
  "label": "8x8",
  "type": "eightbyeightsimple",
}
`;
